.gitattributes

```diff
@@ -1,6 +1,6 @@
 # Ignore vendored code when calculating repo code %s
 # libsnark is lots of c++
-src/lib/snarky/src/camlsnark_c/libsnark-caml/* linguist-vendored
+automation/* linguist-vendored
```

Comment:
Shouldn't you re-add the repository files after `.gitattributes` gets updated? For example:
https://stackoverflow.com/questions/48642692/how-to-make-change-to-gitattributes-take-effect

Reply:
Thanks for pointing that out, I looked up the `linguist-vendored` attribute, it all makes sense now.